.github/workflows/ci.yml

```diff
@@ -48,79 +48,35 @@ jobs:
       - name: make test
         run: make test
 
-  basic_clang:
+  out-of-tree:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
+      - name: setup build dir
+        run: |
+            set -eux
+            mysrcdir=$(realpath .)
+            echo "mysrcdir=$mysrcdir" | tee -a $GITHUB_ENV
+            myblddir=../_build/nest/a/little/more
+            mkdir -p $myblddir
+            echo "myblddir=$myblddir" | tee -a $GITHUB_ENV
       - name: config
-        run: CC=clang ./config --strict-warnings && perl configdata.pm --dump
+        run: |
+            set -eux
+            myblddir=${{ env.myblddir }}
+            mysrcdir=${{ env.mysrcdir }}
```

Comment:
Incidently, this was interesting to see.  I had no idea there was such a feature.
(and no, I don't need to have it explained, I got the connection with `$GITHUB_ENV`)

Reply:
I discovered this today as well!